scripts/competitiond.py

```diff
@@ -708,7 +708,9 @@ def generate_leaderboard(self, num_total_submissions, num_period_submissions):
 
         # Sort by the scores, descending
         leaderboard.sort(
-            key=lambda e: tuple(e['scores'][spec['name']] for spec in self.config['score_specs']),
+            key=lambda e: tuple(
```

Comment:
Yeah, this looks like it will return an array of booleans instead of an array of scores. What was the initial error here?

Reply:
This one: `Traceback (most recent call last): File "competitiond.py", line 848, in <module> main() File "competitiond.py", line 844, in main comp.run_once() File "competitiond.py", line 755, in run_once self.generate_leaderboard(num_total_submissions, num_period_submissions) File "competitiond.py", line 718, in generate_leaderboard reverse=True, TypeError: '<' not supported between instances of 'NoneType' and 'float'` 
Python3 doesn't support this None value comparsion, but Python2 does.